students/minji/users/views.py

```diff
@@ -1,3 +1,54 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data                = json.loads(request.body)
+            email               = data["email"]
+            password            = data["password"]
+            REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            REGEX_PASSWORD = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
```

Comment:
```suggestion
            data                = json.loads(request.body)
            email               = data["email"]
            password            = data["password"]
            
            REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
            REGEX_PASSWORD = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
```

피어리뷰 반영해서 빈줄 삽입 혹은 전체적으로 align 맞춰주셔요~!